app/Console/Command/ProcessDuplicatePlaces.php

```diff
@@ -65,11 +66,17 @@ public function configure(): void
             InputOption::VALUE_NONE,
             'Execute a dry-run of the process-duplicates script.'
         );
+        $this->addArgument(
+            'starting_cluster_id',
+            InputArgument::OPTIONAL,
+            'Enter starting cluster id?'
```

Comment:
This will not be prompted so I would not make it a question but a description. Example how it will be printed:

```
vagrant@udb3:/var/www/udb-silex$ ./bin/udb3.php place:process-duplicates --help
Description:
  Process duplicate places (determine canonical, update event locations and reindex)

Usage:
  place:process-duplicates [options] [--] [<starting_cluster_id>]

Arguments:
  starting_cluster_id   Enter starting cluster id? // <-- here

Options:
  -d, --dry-run         Execute a dry-run of the process-duplicates script.
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```

Also it currently does not give any more information than the name of the argument, so you could also just omit it.